nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/jcamp/JCAMPData.java

```diff
@@ -888,7 +888,8 @@ public Dataset toDataset(String datasetName) throws IOException, DatasetExceptio
             datasetName = suggestName(fpath.toFile());
         }
         // Create a dataset in memory
-        Dataset dataset = new Dataset(datasetName + ".nv", dimSizes, true);
+        Dataset dataset = new Dataset(datasetName, dimSizes, true);
+        dataset.setFile(file);
```

Comment:
setting the File even though the dataset is in memory mode so that when the project is saved it knows where to link to.

Reply:
I removed it because when the project was loaded the datasetName passed into dataset contains the .dx, so the filename ends up being <filename text>.dx.nv and then it was not recognized as the correct dataset, I need to take a closer look at this because it seems a bit sketchy for that to have solved the problem, and this probably isn't the best solution

Reply:
added the dataset file as an option in the constructor to make it match better with the other Dataset constructors